lib/client/access-token.js

```diff
@@ -12,7 +14,15 @@ module.exports = function (config) {
 
   function AccessToken(token) {
     this.token = token;
-    this.token.expires_at = addSeconds(new Date(), token.expires_in);
+    if ({}.hasOwnProperty.call(this.token, 'expires_at')) {
+      if (!isDate(this.token.expires_at)) {
+        this.token.expires_at = parse(this.token.expires_at);
+      }
+    } else {
+      this.token.expires_at = addSeconds(new Date(), token.expires_in);
+    }
+
+    return this;
```

Comment:
Should I remove this line? It was part of the old code, I just noticed that you removed it on the latest develop commits.